src/app/templates/partials/breadcrumbs.ftl

```diff
@@ -0,0 +1,12 @@
+<div class="breadcrumbs">
+    <div class="pl-md-1"></div>
+    <#list breadcrumbs as crumb>
+        <div class="breadcrumb-item">
+            <#if crumb.url?has_content>
+                <a href="${crumb.url}">${crumb.name}</a>
+            <#else>
```

Comment:
I'm not really sure how useful an unclickable breadcrumb that just repeats the error message on the page actually is. It looks a bit strange. Maybe including a 'get me out of here' link back to the homepage would be better, either in the main error/404 page content, or as a breadcrumb preceding the unclickable one. 

Also the unclickable breadcrumbs look exactly like the ones which are links, might be better to distinguish them.

Reply:
Oh, that was what that `color: black` style we just deleted was for :woman_facepalming:

Reply:
Ok, I've left the unclickable breadcrumb there but added the main menu as a parent crumb, to be consistent with current behaviour, e.g. https://montagu.vaccineimpact.org/reports/modup2-201810/20190416-134824-a06bab9c/whatever

Also realised `breadcrumb-item` is a built in bootstrap class, so have renamed ours to `crumb-item` to avoid inheriting unwanted styles, and made the unclickable one the regular font color to distinguish from links.